client/app/TagList/components/TagList.jsx

```diff
@@ -1,18 +1,26 @@
 import React, { PropTypes } from 'react'
 import Tag from './Tag'
 
-function TagList(props) {
+function TagList({tags, summary}) {
+  const tagList = summary ? tags.slice(0, 3) : tags
+  const showEllipsis = summary && tags.length > tagList.length
+  const ellipsis = showEllipsis ? (
+    <span className="c-tag-list__ellipsis">…</span>
+  ) : null
+
   return (
-    <ul className="c-tag-list o-list-inline">
-      {props.tags.map(tag =>
+    <ul className={`o-list-inline c-tag-list${ summary ? ' c-tag-list--summary' : ''}`}>
```

Comment:
@marcaltmann Da du ja weiter oben auch schon variablen mit summary-switch eingeführt hast, hätte ich das eher auch in eine summaryClassName variable gepackt